meganav.yaml

```
@@ -1,3 +1,699 @@
+          links:
+            - title: Read documentation
+              description: Official Ubuntu Documentation
+              url: https://help.ubuntu.com/
+            - title: Read the blogs
+              description: Blogs from Canonical
+              url: /blog
+            - title: Engage pages
+              description: Support literature hub
+              url: /engage
+            - title: Tutorials
+              description: Guides for development activities
+              url: /tutorials/
+            - title: Ask Ubuntu
+              description: Q&A site for users and developers
+              url: https://askubuntu.com/
+            - title: Ubuntu Certified Hardware
+              description: Hardware you can trust
+              url: /certified
+            - title: About Ubuntu
+              description: Introduction to the OS
+              url: /about
+            - title: Ubuntu forums
+              description: Discuss Ubuntu with other users
+              url: https://ubuntuforums.org/
+      secondary_links:
+        - links:
+            - title: Webinars
+              url: /engage?resource=webinar
+            - title: Case studies
+              url: /engage?resource=case+study
+            - title: Whitepapers
+              url: /engage?resource=whitepaper
+            - title: Become an Ubuntu certified engineer
+              url: /cube
+            - title: Videos
+              url: https://www.youtube.com/c/UbuntuOS
+
+community:
+  side_nav_sections:
+    - title: Learning resources
+      primary_links:
+        - title: Resources
+          links:
+            - title: Tutorials
+              description: Guides for development activities
+              url: /tutorials/
+            - title: Documentation
+              description: Official Ubuntu Documentation
+              url: https://help.ubuntu.com/
+            - title: UbuntuOnAir
+              description: Community-focused videos
+              url: https://www.youtube.com/ubuntuonair
+            - title: Discourse
+              description: Coordinating the Ubuntu project
+              url: https://discourse.ubuntu.com/
+            - title: Ask Ubuntu
+              description: Q&A for users and developers
+              url: https://askubuntu.com/
+            - title: Forums
+              description: Discuss Ubuntu with other users
+              url: https://ubuntuforums.org/
+      secondary_links:
+        - title: Links
+          links:
+            - title: Update Ubuntu
+              url: /tutorials/upgrading-ubuntu-desktop#1-before-you-start
+            - title: Official Ubuntu documentation
+              url: https://help.ubuntu.com/
+            - title: Install guide
+              url: /tutorials/install-ubuntu-desktop#1-overview
+            - title: How to write a tutorial
+              url: /tutorials/tutorial-guidelines#1-overview
+    - title: Forums
+      primary_links:
+        - title: Resources
```

```suggestion
        - title: Forums
```
How it appears in the spreadsheet